solana/solana-ibc/programs/solana-ibc/src/transfer/impls.rs

```diff
@@ -108,3 +221,21 @@ impl TokenTransferValidationContext for IbcStorage<'_, '_> {
         Ok(())
     }
 }
+
+fn get_account_info_from_key<'a, 'b>(accounts: &'a Vec<AccountInfo<'b>>, key: Pubkey) -> Result<&'a AccountInfo<'b>, TokenTransferError> {
```

Comment:
Prefer taking slice arguments rather then reference to vector.

```suggestion
fn get_account_info_from_key<'a, 'b>(accounts: &'a [AccountInfo<'b>], key: Pubkey) -> Result<&'a AccountInfo<'b>, TokenTransferError> {
```